server/src/main/java/pl/bk/pizza/store/domain/discount/DiscountPriority.java

```diff
@@ -0,0 +1,16 @@
+package pl.bk.pizza.store.domain.discount;
+
+import com.google.common.collect.ImmutableSortedMap;
+import pl.bk.pizza.store.domain.discount.bonus.ExtraProductDiscount;
+import pl.bk.pizza.store.domain.discount.bonus.PercentDiscount;
+import pl.bk.pizza.store.domain.discount.bonus.TotalPriceDiscount;
+
+public class DiscountPriority
+{
+    public static ImmutableSortedMap<Integer, Class<? extends Discount>> prioritizeDiscounts =
```

Comment:
\[Checkstyle\] ERROR: Variable 'prioritizeDiscounts' must be private and have accessor methods\.